functests/sctp/sctp.go

```diff
@@ -483,3 +488,59 @@ func setupEgress(namespace, fromPod, toPod string, port int32) error {
 	_, err := client.Client.NetworkPolicies(namespace).Create(policy)
 	return err
 }
+
+func findSCTPNodeSelector() {
```

Comment:
Can we have this return the selector or an error if not found, and use the result in `beforeAll`?